Package.swift

```diff
@@ -1,11 +1,12 @@
-// swift-tools-version: 5.6
+// swift-tools-version:5.9
 // The swift-tools-version declares the minimum version of Swift required to build this package.
 
+import CompilerPluginSupport
 import PackageDescription
 
 let package = Package(
     name: "klaviyo-swift-sdk",
-    platforms: [.iOS(.v13)],
+    platforms: [.iOS(.v15), .macOS(.v10_15)],
```

Comment:
Might not want macOS but also maybe it will just work.

Reply:
So people have run the ios app on a mac but I'm not sure if that's the same thing.

Reply:
I ended up taking this out. We need to do a little more work to support this.